grouped-categories.js

```diff
@@ -623,4 +623,37 @@
 		}
 	});
 
+	// Override `fontMetrics` function to provide compatibility with HC 11 (#200)
+	HC.SVGRenderer.prototype.fontMetrics = function (fontSize, elem) {
```

Comment:
Instead of overriding `fontMetrics` on Highcharts, create a local function used internally when `fontMetrics` is needed.